rootfs-builder/rootfs.sh

```diff
@@ -163,17 +168,32 @@ shift $(($OPTIND - 1))
 [ -n "${KERNEL_MODULES_DIR}" ] && [ ! -d "${KERNEL_MODULES_DIR}" ] && die "KERNEL_MODULES_DIR defined but is not an existing directory"
 
 distro="$1"
-init="${ROOTFS_DIR}/sbin/init"
 
 [ -n "${distro}" ] || usage 1
 distro_config_dir="${script_dir}/${distro}"
 
-[ -d "${distro_config_dir}" ] || die "Not found configuration directory ${distro_config_dir}"
-rootfs_lib="${distro_config_dir}/${LIB_SH}"
-source "${rootfs_lib}"
-rootfs_config="${distro_config_dir}/config.sh"
+# Source config.sh from distro
+rootfs_config="${distro_config_dir}/${CONFIG_SH}"
 source "${rootfs_config}"
 
+lib_file="${script_dir}/../scripts/lib.sh"
+info "Source $lib_file"
+[ -e "$lib_file" ] && source "$lib_file" || true
```

Comment:
probably would be better to check if is a regular file `-f`, could be modified later.